client/a8c-for-agencies/sections/marketplace/products-overview/product-card/index.tsx

```diff
@@ -143,6 +143,7 @@ export default function ProductCard( props: Props ) {
 							<Button
 								className="product-card__select-button"
 								variant={ isSelected ? 'secondary' : 'primary' }
+								tabIndex={ -1 }
```

Comment:
Same as above, we don't need to make this button tabbable so as not to duplicate the card's behavior.